odhargo/base/workflow-crd.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Should the CRDs go into a "cluster" dir?

Reply:
@crobby is that what we are doing? If so I can move them

Reply:
Yeah, anything that would be a "cluster-admin" sort of action goes to "cluster".